admin/app/components/solidus_admin/stores/edit/component.html.erb

```diff
@@ -0,0 +1,87 @@
+<%= page do %>
+  <%= page_header do %>
+    <%= page_header_back(solidus_admin.stores_path) %>
+    <%= page_header_title(t(".title", store: @store&.name)) %>
+    <%= page_header_actions do %>
+      <div class="py-1.5 text-center">
+        <%= render component("ui/button").new(tag: :button, text: t(".update"), form: form_id) %>
+        <%= render component("ui/button").new(tag: :a, text: t(".cancel"), href: solidus_admin.edit_store_path(@store), scheme: :secondary) %>
+      </div>
+    <% end %>
+  <% end %>
+
+  <%= form_for @store, url: solidus_admin.store_path(@store), html: { id: form_id } do |f| %>
+    <%= page_with_sidebar do %>
+      <%= page_with_sidebar_main do %>
+        <%= render component("ui/panel").new(title: t(".store_settings")) do %>
```

Comment:
Why haven't you used the existing address form component?

Reply:
It didn't seem indicated to invoke the address list for performance reasons. 
These values should be cached and provided separately as otherwise the address tabled needs to be used for values that in some scenarios could be embedded on every page depending on the implementation.